spec/views/content_items/show.html.erb_spec.rb

```diff
@@ -1,12 +1,10 @@
 require 'rails_helper'
 
 RSpec.describe 'content_items/show.html.erb', type: :view do
-  let(:content_item) { build(:content_item) }
-  let(:organisation) { build(:organisation) }
+  let(:content_item) { create(:content_items_with_org, organisations_count: 2) }
```

Comment:
I would prefer to stick to 1 organisation per content item, and only use 2 when we are checking the way we render organisations. The tests would be more expressive, and we they will also perform better.